# HG changeset patch
# User Rik <rik@octave.org>
# Date 1381529930 25200
#      Fri Oct 11 15:18:50 2013 -0700
# Node ID cd79bb815eb1c5ef8f19b2923317b74f4db2ce46
# Parent  b30bb9bb05c8a88d2bd0a1fc831a29d7a9bba7fa
maint: Remove unnecessary function provoking -Wunused-function warning.

* liboctave/util/data-conv.cc(gripe_data_conversion): Remove unused function.

diff --git a/liboctave/util/data-conv.cc b/liboctave/util/data-conv.cc
--- a/liboctave/util/data-conv.cc
+++ b/liboctave/util/data-conv.cc
@@ -622,23 +622,16 @@ oct_data_conv::data_type_as_string (oct_
 
 static void
 gripe_unrecognized_float_fmt (void)
 {
   (*current_liboctave_error_handler)
     ("unrecognized floating point format requested");
 }
 
-static void
-gripe_data_conversion (const char *from, const char *to)
-{
-  (*current_liboctave_error_handler)
-    ("unable to convert from %s to %s format", from, to);
-}
-
 // But first, some data conversion routines.
 
 // Currently, we only handle conversions for the IEEE types.  To fix
 // that, make more of the following routines work.
 
 // FIXME -- assumes sizeof (Complex) == 8
 // FIXME -- assumes sizeof (double) == 8
 // FIXME -- assumes sizeof (float) == 4
